lib/db/schema/patch-078-079.sql

```diff
@@ -0,0 +1,57 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+ALTER TABLE recoveryCodes MODIFY COLUMN codeHash BINARY(32),
+ALGORITHM = COPY, LOCK = SHARED;
```

Comment:
When I attempted to use `ALGORITHM=INPLACE, LOCK=NONE`, mysql errored with 

```
Error: ER_ALTER_OPERATION_NOT_SUPPORTED_REASON: ALGORITHM=INPLACE is not supported. Reason: Cannot change column type INPLACE. Try ALGORITHM=COPY.
```

This alter is changing the column from `BINARY(64)` to `BINARY(32)` since that is what password hashes are stored as. At first glance, I don't see any issues using `ALGORITHM=COPY` (or none at all). Mysql docs says that `COPY` is used on all alters if `INLINE` can't be used. Is there something I else I might be missing here?

@rfk @philbooth Thoughts?